docs/site/Validation-REST-layer.md

```diff
@@ -175,3 +175,48 @@ of 10 digits separated by `-` after the 3rd and 6th digits.
 ```
 
 {% include tip.html content="RegExp can be converted into a string with `.source` to avoid escaping backslashes" %}
+
+## Customize validation errors
+
+Since the error is being caught at the REST layer, the best way to customize the
+errors is to
+[customize the sequence of actions](https://loopback.io/doc/en/lb4/Sequence.html#customizing-sequence-actions)
+using a
+[Provider](https://loopback.io/doc/en/lb4/apidocs.context.provider.html).
+
+To do so, create a Provider class in `src/providers` folder that implements
+`Provider<Reject>`. Using the
+[validation-app](https://github.com/strongloop/loopback-next/tree/master/examples/validation-app)
+as an example, here is how the Provider class look like:
```

Comment:
> here is how the Provider class look like

here is how the Provider class looks like

Reply:
I don't think this part was updated